src/userbase-server/setup.js

```diff
@@ -334,6 +337,21 @@ async function createTable(ddb, params) {
   }
 
   await ddb.waitFor('tableExists', { TableName: params.TableName, $waiter: { delay: 2, maxAttempts: 60 } }).promise()
+
+  const enableBackup = async function () {
+    try {
+      await ddb.updateContinuousBackups({ TableName: params.TableName, PointInTimeRecoverySpecification: { PointInTimeRecoveryEnabled: true } }).promise()
+    } catch (e) {
+      if (e.code === 'ContinuousBackupsUnavailableException') {
+        setTimeout(enableBackup, 15000)
+      } else {
+        logger.error(e)
+        throw e
+      }
+    }
+  }
+
+  enableBackup()
```

Comment:
It was done like this because it takes a long time for the backups to get enabled after a table gets created. Sometimes more than 10 minutes. If we block, then starting locally can take forever if a table needs to be created.

In prod, an attempt to enable backups will be made after every deployment, and if the table had been created for a while, then it turns on instantly.